src/style.css

```diff
@@ -0,0 +1,113 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+/* @font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+} */
+
+/* html {
+  font-family: "Avenir", Arial, sans-serif;
+} */
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-weight: 300px;
+}
+
+.form-big {
+  position: relative;
+  min-width: 200vh;
+  margin-top: 20px;
+  margin-right: 8px;
+  margin-left: 8px;
+  background-position: center;
+}
+
+.img-big {
+  position: absolute;
+  padding-left: 26px;
+  top: 25px;
+  bottom: 26px;
+  height: 19px;
+  width: 19px;
+}
+
+.input-big::placeholder {
+  color: #3d4e61;
+}
+
+.input-big {
+  width: 100%;
+  height: 70px;
+  display: flex;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+}
+
+.input-big:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input-big:focus {
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
```

Comment:
why do we need to change it on hover?

Reply:
You show the focus on this part. If you wanted to ask why change the search when hovering, I performed the task according to the layout and there when hovering the search gets a shadow